src/connector/src/sink/iceberg.rs

```diff
@@ -58,47 +55,33 @@ pub type RemoteIcebergConfig = RemoteConfig;
 #[serde(deny_unknown_fields)]
 pub struct IcebergConfig {
     #[serde(skip_serializing)]
-    pub connector: String, // Must be "kafka" here.
+    pub connector: String, // Must be "iceberg" here.
 
     pub r#type: String, // accept "append-only" or "upsert"
 
     #[serde(default, deserialize_with = "deserialize_bool_from_string")]
     pub force_append_only: bool,
 
-    #[serde(rename = "warehouse.path")]
-    pub path: String,
-
-    #[serde(rename = "s3.region")]
-    pub region: Option<String>,
-
-    #[serde(rename = "s3.endpoint")]
-    pub endpoint: Option<String>,
-
-    #[serde(rename = "s3.access.key")]
-    pub access_key: String,
-
-    #[serde(rename = "s3.secret.key")]
-    pub secret_key: String,
-
-    #[serde(rename = "database.name")]
-    pub database_name: String,
-
     #[serde(rename = "table.name")]
-    pub table_name: String,
+    pub table_name: String, // Full name of table, must include schema name
 
     #[serde(skip)]
-    pub iceberg_table_config: TableConfigRef,
+    pub iceberg_configs: HashMap<String, String>,
```

Comment:
but using HashMap will collect props not starts with `iceberg`

Reply:
It's not a big problem. `icelake` will check names  and we also did filtering in risingwave.